ibis/omniscidb/tests/test_client.py

```diff
@@ -178,8 +178,8 @@ def test_explain(con, alltypes):
 @pytest.mark.parametrize(
     'filename',
     [
-        "/omnisci/test_read_csv.csv",
-        pathlib.Path("/omnisci/test_read_csv.csv"),
+        "/tmp/test_read_csv.csv",
```

Comment:
can you use a temporary filename instead here

Reply:
so what you should do is just put the filename itself in the parameterize, then use a temporary filename inside

Reply:
I'm not sure that i got your idea, should i do like this?
```
@pytest.mark.paratetrize(
    'filename',
    [
        "test_read_csv.csv",
        pathlib.Path("test_read_csv.csv"),
    ]
)
```
Well, then we make path relative to the `omnisci_server` executable, and for some reason at that case it creates all export files at `<OMNISCI_DATAPATH>/mapd_export/<SESSION_ID>/filename`.
So `COPY (SELECT * FROM functional_alltypes) TO 'filename'` will create file at that path,
but `COPY functional_alltypes FROM 'filename'` at `read_csv` will try to read file directly at `filename` path (which is relative to omnisci executable) and get an error, because that file does not exist there.

Reply:
If you suggested to use [python tempfile](https://docs.python.org/3/library/tempfile.html), then no, i can't use it.
Because, as i said at the above message, we pass `filename` as a file path on the server, and it could be a separated machine from which we are running our test script.

Reply:
@dchigarev this is a test; you certainly can use a temp file in any location.

Reply:
But i thought `tempfile` is just for managing temporary files at local machine where scripts are executing?
We may have running test scripts and omnisci at two different machines, so file generated by `tempfile` by test script, will not be valid for where omnisci is running?